plugin-machine/che-plugin-machine-ext-client/src/main/resources/org/eclipse/che/ide/extension/machine/client/machine.css

```diff
@@ -31,6 +31,7 @@
 
     color: outputFontColor;
     line-height: 13px;
+    padding: 3px;
```

Comment:
As it discussed with Stevan, padding was removed and determining the terminal size was fixed for mac.
Can you remove this line?